scripts/validate-changelog.sh

```diff
@@ -11,6 +11,11 @@ package_name="$1"
 
 if [[ "${GITHUB_REF:-}" =~ '^release/' ]]; then
   yarn auto-changelog validate --tag-prefix "${package_name}@" --rc
+elif [[ $# -gt 2 ]]; then
+  # In case of package rename
```

Comment:
It looks like we are only taking in the version and tag prefix if we are not validating a release branch. However, we will still want to do this, because we may still be validating a changelog where the package was renamed in the past.

It looks like we are passing the version and tag prefix as the second and third arguments. What if we have this script take the package name as the only explicit argument, and then slurp up the remaining arguments into an array and pass them all to `auto-changelog`? This way we don't ever have to change this script if we add a new option to `auto-changelog`.

We could do something like this:

``` bash
package_name="$1"
shift  # remove package name from arguments

if [[ "${GITHUB_REF:-}" =~ '^release/' ]]; then
  yarn auto-changelog validate --tag-prefix "${package_name}@" --rc "$@"
else
  yarn auto-changelog validate --tag-prefix "${package_name}@" "$@"
fi